trace/cloud-trace-demo-app/app/requirements.txt

```diff
@@ -0,0 +1,10 @@
+google-cloud-monitoring==0.33.0
+google-cloud-trace==0.23.0
+opencensus==0.7.5
+Flask==1.1.1
+opencensus-ext-stackdriver==0.7.2
+opencensus-ext-flask==0.7.3
+opencensus-context==0.1.1
+grpcio==1.25.0
+six==1.13.0
```

Comment:
I didn't see six being used above - are you sure it's a requirement?

Reply:
Good catch. I removed "six" dependency.